snippets/knowledgebase/static and dynamic rendering.md

```diff
@@ -19,4 +19,7 @@ Dynamic data fetching options:
 - cache: `no-store`
 - revalidate: `0`
 
+Dynamic secment configurations:
+- dynamic = `force-dynamic`
```

Comment:
```suggestion
- `export const dynamic = 'force-dynamic'`
```

Reply:
This is to show the  the configurations that can be specified, not an actual code example, like the ones above